apollo-server/v4/generated-types/src/datasources.ts

```diff
@@ -0,0 +1,37 @@
+// Use our automatically generated Book and AddBookMutationResponse types
+// for type safety in our data source class
+import { AddBookMutationResponse, Book } from './__generated__/resolvers-types';
+
+export class BooksDataSource {
+// Our example static data set
+  books: {title?: string, author?:string}[] = [
```

Comment:
this does not look prettier to me

Reply:
Ooops, forgot to turn that on —thank you!